// 예외를 발생시키고 예외 처리하지 않았을 때 결과를 보는 예제
/*public class ExceptionEx10 {
    public static void main(String[] args) {
        throw new Exception();
    } // 예외처리가 되어 있지 않다는 에러가 발생
}*/
